cmd/shipperctl/main.go

```diff
@@ -18,10 +21,11 @@ var rootCmd = &cobra.Command{
 }
 
 func init() {
-	rootCmd.AddCommand(cmd.ClustersCmd)
-	rootCmd.AddCommand(cmd.ListCmd)
-	rootCmd.AddCommand(cmd.CleanCmd)
+	rootCmd.AddCommand(clusters.ClustersCmd)
+	rootCmd.AddCommand(list.ListCmd)
+	rootCmd.AddCommand(clean.CleanCmd)
```

Comment:
nitI really like the split into packages, maybe I would rename `*Cmd` into `Cmd` to remove the stutter.